### PR TITLE
Enable eslint for cloudflare package

### LINKED ISSUE #30: ESlint rules to add
- enforce `node:` prefix
- sort imports

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
fixes #30

### CODE REVIEW

**Review by vicb — APPROVED** [review]
Looks great, thanks!

File: .github/workflows/prettier.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Prettier check
+name: Code checks
```
[the remark this answers is not in the document]
Comment (dario-piotrowicz) [reply .github/workflows/prettier.yml:1]: oh! good point sorry I missed that! 🙂

[the remark this answers is not in the document]
Comment (dario-piotrowicz) [reply .github/workflows/prettier.yml:1]: done 🙂

File: package.json [hunk 2]
```diff
@@ -9,6 +9,8 @@
   "scripts": {
     "prettier:check": "prettier --check .",
     "prettier:fix": "prettier --write .",
+    "lint:check": "pnpm -r lint:check",
+    "lint:fix": "pnpm -r lint:fix",
```
[the remark this answers is not in the document]
Comment (dario-piotrowicz) [reply package.json:13]: I've added this: https://github.com/opennextjs/opennextjs-cloudflare/pull/44/commits/9ec72f94729c5b52768962babed93b2d41ac2704

Please let me know if this is what you had in mind 🙂

File: package.json [hunk 3]
```diff
@@ -9,6 +9,10 @@
...
+    "code:checks": "pnpm lint:check ; pnpm lint:check",
```
Comment (vicb) [inline package.json:14]:
```suggestion
    "code:checks": "pnpm lint:check && pnpm lint:check",
```

Comment (dario-piotrowicz) [reply package.json:14]: I used `;` so that you can see all the failures instead of stopping to the lint ones, but if you prefer I'm happy with `&&` 🙂

Comment (vicb) [reply package.json]: Oh nice... TIL

**Pull request merged**